src/doc/DocDrawingDialog.js

```diff
@@ -145,7 +146,7 @@ class DocDrawingDialog extends AnnotationDialog {
 
         const firstAnnotation = util.getFirstAnnotation(annotations);
         if (firstAnnotation) {
-            this.assignDrawingLabel(firstAnnotation);
+            this.addAnnotation(firstAnnotation);
```

Comment:
Does addAnnotation get called when creating an annotation OR rendering an existing one? 

My understanding of this code is:
We're setting up a dialog, so if there's an annotation already in that dialog, add it to the dialog. Draw dialogs can only have one annotation (no comments), so we're not really adding an annotation here, but loading the previously saved annotation into this dialog.

Reply:
Both. addAnnotation just adds the element to the dialog so it occurs either when the dialog is initially being set up or when a new annotation is saved to the thread (which only occurs once for drawing annotations)